python/lsst/meas/algorithms/brightStarStamps.py

```diff
@@ -189,17 +193,29 @@ class BrightStarStamps(Stamps):
         stamps, stamps normalized with different annulus definitions, or if
         stamps are to be normalized but annular radii were not provided.
 
-
     Notes
     -----
     A butler can be used to read only a part of the stamps, specified by a
     bbox:
 
-    >>> starSubregions = butler.get("brightStarStamps", dataId, parameters={'bbox': bbox})
+    >>> starSubregions = butler.get(
+            "brightStarStamps",
+            dataId,
+            parameters={"bbox": bbox}
+        )
     """
 
-    def __init__(self, starStamps, innerRadius=None, outerRadius=None, nb90Rots=None,
-                 metadata=None, use_mask=True, use_variance=False, use_archive=False):
+    def __init__(
```

Comment:
I find the placement of this weird. I'd like it either before or immediately after the `factory` method above.

Reply:
This is the `__init__` for the BrightStarStamps class, and the first thing in this class - did you have in mind somewhere else specifically for this to go?

Reply:
Unless you're referring to the `__len__`, `__getitem__` and `__iter__` methods below, which could all be moved up?

Reply:
Thanks for the in-person chat about this Arun. Following our chat, I think we've agreed that we're happy leaving this `__init__` as-is for now.